certified-connectors/Easyvista_Service._Manager/Readme.md

```diff
@@ -0,0 +1,277 @@
+Connector Easyvista Service Manager
```

Comment:
@ezv-compteintegration Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for the readme file. You can see the raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Reply:
@ezv-compteintegration Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for the readme file. You can see the raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).